big-o/big-o-notation.md

```diff
@@ -107,10 +122,128 @@ Consider this problem: You have an address book, and you want to find the contac
 
 <!-- Question about comparing Big Os for more efficient -->
 
-<!-- Question about comparing Big Os for more efficient -->
+<!-- >>>>>>>>>>>>>>>>>>>>>> BEGIN CHALLENGE >>>>>>>>>>>>>>>>>>>>>> -->
+<!-- Replace everything in square brackets [] and remove brackets  -->
+
+### !challenge
+
+* type: ordering
+* id: d2d156f6-aaab-4aba-9cf7-b9cdaceed013
+* title: Time/Space Complexities Worst-to-best
+* points: 1
+* topics: big-o
+
+##### !question
+
+Arrange the following complexities from **worst** to **best**.
+
+##### !end-question
+
+##### !answer
+
+1. O(2^n)
+1. O(n^2)
```

Comment:
Just one more thought that when these were introduced, you used the nice superscript stuff.  Maybe do that here?  Or introduce the more computer-y style of writing it above as well?

Reply:
Ok I've switched both to the `<sub>` notation.  I think the original lesson here came from two different sources which is why the discrepancy.